src/helpers/signatureHelper.ts

```diff
@@ -0,0 +1,99 @@
+import { Request, Response, NextFunction } from 'express';
+import crypto from 'crypto';
+import env from './env';
+
+const wellKnownUpSignOnPublicKey = env.IS_PRODUCTION
+  ? '-----BEGIN PUBLIC KEY-----\nMFYwEAYHKoZIzj0CAQYFK4EEAAoDQgAEB/4zGfZRH715/kObqaWDL2nZR+ybUh/y\nx/wCHiLXX9VDzSh/qB+NC0KJ/rMPKXKhCCKgWCo/OBxieXarHJS6lg==\n-----END PUBLIC KEY-----\n'
```

Comment:
this one should be part of a .env, and the private key should be part of a vault. where is this privatekey?